config/default/kustomizeconfig.yaml

```diff
@@ -0,0 +1,4 @@
+namePrefix:
+- kind: Deployment
+  name: controller-manager
+  path: spec/template/spec/volumes/secret/secretName
```

Comment:
What is the difference between this `namePrefix` and the one in `config/certificate/kustomization.yaml`?

Reply:
The directive in `kustomization.yaml` specifies the value of the prefix. This specifies which fields need to be specified.